src/pages/Interferometry/MachZehnderEquipment.js

```diff
@@ -117,21 +117,26 @@ class MachZehnderEquipment {
 
   fire(context, photon, photonFire, probability) {
     if (photonFire) {
+      // photon movement
+      // decide position of photon
       photon.move(this.detector0, this.mirror0, this.bs1, this.bs2, probability);
+      // draw photon particle
```

Comment:
we also don't need this comment. it's not informative.

Reply:
removed!